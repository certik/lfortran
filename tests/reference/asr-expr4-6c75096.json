{
    "basename": "asr-expr4-6c75096",
    "cmd": "lfortran --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/expr4.f90",
    "infile_hash": "104847f7b62623d6e9d6d752ab2c6c58a31b70bc70a9cae0b5809241",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "asr-expr4-6c75096.stdout",
    "stdout_hash": "3aa5029aadb8717a0f344c22c6a7609c69bc8880499fdd7f63e96b4f",
    "stderr": null,
    "stderr_hash": null,
    "returncode": 0
}