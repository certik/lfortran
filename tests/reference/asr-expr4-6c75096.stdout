(TranslationUnit (SymbolTable 1 {}) [(RealConstant   5.00000000000000000e+00 (Real 4 [])) (RealBinOp (RealConstant   5.00000000000000000e+00 (Real 4 [])) Add (RealConstant   3.00000000000000000e+00 (Real 4 [])) (Real 4 []) (RealConstant   8.00000000000000000e+00 (Real 4 []))) (RealBinOp (RealBinOp (RealConstant   5.00000000000000000e+00 (Real 4 [])) Add (RealConstant   3.00000000000000000e+00 (Real 4 [])) (Real 4 []) (RealConstant   8.00000000000000000e+00 (Real 4 []))) Mul (RealConstant   2.00000000000000000e+00 (Real 4 [])) (Real 4 []) (RealConstant   1.60000000000000000e+01 (Real 4 []))) (RealBinOp (RealConstant   5.00000000000000000e+00 (Real 4 [])) Add (RealBinOp (RealConstant   3.00000000000000000e+00 (Real 4 [])) Mul (RealConstant   2.00000000000000000e+00 (Real 4 [])) (Real 4 []) (RealConstant   6.00000000000000000e+00 (Real 4 []))) (Real 4 []) (RealConstant   1.10000000000000000e+01 (Real 4 []))) (RealBinOp (RealConstant   5.00000000000000000e+00 (Real 4 [])) Sub (RealConstant   3.00000000000000000e+00 (Real 4 [])) (Real 4 []) (RealConstant   2.00000000000000000e+00 (Real 4 []))) (RealBinOp (RealConstant   4.00000000000000000e+00 (Real 4 [])) Pow (RealConstant   3.00000000000000000e+00 (Real 4 [])) (Real 4 []) (RealConstant   6.40000000000000000e+01 (Real 4 [])))])
